.github/workflows/release.yml

```diff
@@ -21,47 +21,12 @@ jobs:
     - name: Check targets are installed correctly
       run: rustup target list --installed
 
-    - uses: actions/cache@v3
-      with:
-        path: |
-          ~/.cargo/registry
-          ~/.cargo/git
-          target/
-        key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-cargo-
-
     - name: Check all features compilation
       run: cargo check --verbose --all-features
 
     - name: Run all tests
       run: cargo test --all-features
 
-  native-linux:
-    needs: checks-and-tests
-    runs-on: [self-hosted, Linux, X64]
```

Comment:
Do we need to remove cache for selfhosted runner as well?

Reply:
> Do we need to remove cache for selfhosted runner as well? If you look at this run https://github.com/AstarNetwork/Astar/runs/7410118550?check_suite_focus=true, self hosted runner exist without error when cache size exceeds 10GB.

It isn't red but it failed.
Please check the Post Run Action.
You can see that the cached archive is close to 13 GB in size so the upload fails.

The cache action documentation specifies that we github only allows 10 GiB of space for cache in total.
And with the size we're going on right now, this isn't enough to store one archive.